pages/audit/overview.mdx

```diff
@@ -1,21 +1,17 @@
----
-layout: 'guide'
-title: 'What is a Tracking Plan audit?'
-#abstract: 'Learn how to use Avo to identify issues in your existing tracking plan'
----
+# What is a Tracking Plan audit?
 
-import Link from '../../components/Link';
+_Learn how to use Avo to identify issues in your existing tracking plan_
 
 The Avo tracking plan audit reviews your event and property definitions and flags issues such as unexpected naming or inconsistent types — helping you keep your tracking plan clean and consistent.
 
 The tracking plan audit monitors every change you make in your Avo Tracking Plan and highlights if new issues arise. You can think about the tracking plan audit like a spell-checker for your tracking plan.
 
-Furthermore, all changes on branches are also checked in the <Link passHref href="/audit/branch-audit"><a>Branch Audit</a></Link> to see if you have introduced any new issues on your branch.
+Furthermore, all changes on branches are also checked in the [Branch Audit](/audit/branch-audit) to see if you have introduced any new issues on your branch.
 
-To learn more about the audit rules, see <Link passHref href="/audit/rules"><a>Avo's tracking plan audit rules</a></Link>.
+To learn more about the audit rules, see [Avo's tracking plan audit rules](/audit/rules).
 
-> **Note:** The tracking plan audit only audits the _plan_. If you want to identify issues in your current tracking implementation and data streams, check out <Link passHref href="/workspace/inspector"><a>Tracking Observability</a></Link>.
+> **Note:** The tracking plan audit only audits the _plan_. If you want to identify issues in your current tracking implementation and data streams, check out [Tracking Observability](/workspace/inspector).
```

Comment:
You could also use the `Callout` component here. It’s a bit more nice.

Reply:
Hey nice!